specs/core/0_beacon-chain.md

```diff
@@ -966,6 +998,19 @@ def get_attestation_participants(state: BeaconState,
     return participants
 ```
 
+### `is_power_of_two`
+
+```
+def is_power_of_two(value: int) -> bool:
+    """
+    Check if ``value`` is a power of two integer.
+    """
+    if value == 0:
+        return False
+    else:
+        return 2**int(math.log2(value)) == value
```

Comment:
IMO this is dangerous at least if you do it in python. I would do:

```
def is_power_of_two(value: int) -> bool:
    """
    Check if ``value`` is a power of two integer.
    """
    if value == 0:
        return False
    elif value == 1:
        return True
    else:
        return value % 2 == 0 and is_power_of_two(value // 2)
```

Reply:
Actually never mind, I verified that it works fine for all powers of 2 up to 10000 and I think because of how floating point rounding works it actually should always work. So the existing definition could be ok.